src/messages/ImageSet.cpp

```diff
@@ -58,6 +58,10 @@ const ImagePtr &ImageSet::getImage3() const
 
 const ImagePtr &ImageSet::getImage(float scale) const
 {
+    auto emoteScale = getSettings()->emoteScale.getValue();
+
+    scale += emoteScale - 1.f;
```

Comment:
Shouldn't `scale` be multiplied with `emoteScale`?

Reply:
You're right, that feels like it makes more sense in the emote size transitions with different scales